core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts

```diff
@@ -75,8 +75,8 @@ interface CreatorDashboardDataBackendDict {
   'suggestions_to_review_list': SuggestionBackendDict[];
   'explorations_list': CreatorExplorationSummaryBackendDict[];
   'collections_list': CollectionSummaryBackendDict[];
-  // Here topic summary dicts are optional because they are present in response
-  // only if new structrures are enabled.
+  // Here 'topic_summary_dicts' property is optional because it is present
+  // in response only if new structrures are enabled.
```

Comment:
Here and below: could you please change the backend response to return null if new structures are not enabled, rather than leave out the key-value pair altogether? /cc @aks681

Reply:
done